frontend/src/screens/apps/ShowApp.tsx

```diff
@@ -222,7 +222,7 @@ function AppInternal({ app, refetchApp, capabilities }: AppInternalProps) {
                             type="button"
                             variant="outline"
                             onClick={() => {
-                              window.location.reload();
+                              setEditMode(false);
```

Comment:
No this won't work (try setting the permissions to custom and cancel without saving, you won't see any permissions)

It used to work this way before but @rolznz changed this to window reload because this adds a lot of complexity to the Permissions component (that's why the prop passed ot Permissioins is renamed from `isEditMode` to `readOnly`)

Reply:
We can somehow remove the edit param and then do the reload to fix the issue instead

Reply:
It's just so if they cancel and edit again, it resets the values. But how many people are going to do this and if they edit it incorrectly they can just edit it again

I actually think @reneaaron 's fix is ok?

Reply:
It fixes the issue of not reloading with edit mode again but I think it would feel buggy to see the edited state even after clicking cancel, but yeah the chances of that happening are slim

Reply:
oh, it even applies to what's displayed on the screen, hmm...

Maybe let's leave this open then and re-think it

Reply:
Yeah, but if we want to fix the actual issue for now (#333) we can remove the param value and do the reload

Reply:
I will close this for now - let's do a new PR to fix it